test/e2e_node/device_plugin_failures_test.go

```diff
@@ -352,4 +352,61 @@ var _ = SIGDescribe("Device Plugin Failures:", framework.WithNodeConformance(),
 		// after the graceful period devices capacity will reset to zero
 		gomega.Eventually(getNodeResourceValues, devicePluginGracefulTimeout+1*time.Minute, f.Timeouts.Poll).WithContext(ctx).WithArguments(resourceName).Should(gomega.Equal(ResourceValue{Allocatable: 0, Capacity: 0}))
 	})
+
+	ginkgo.It("when ListAndWatch will fail immediately in shared GPU scenario, node allocatable will be set to zero and kubelet will not retry to list resources", func(ctx context.Context) {
```

Comment:
is this ready for review? Looks like WIP

Reply:
Ready now.